programs/gateway/src/state/voter_weight_record.rs

```diff
@@ -0,0 +1,113 @@
+use anchor_lang::prelude::*;
+
+use crate::tools::anchor::{DISCRIMINATOR_SIZE, PUBKEY_SIZE};
+
+/// The default vote weight matches the default decimal places of a governance token
+/// so that a single vote using this plugin matches a single vote with a governance token
+pub const DEFAULT_VOTE_WEIGHT: u64 = 1000000;
```

Comment:
nit: Due to `Anchor` limitations `voter_weight_record.rs` is copied as is into the plugins code and it's better no not make local changes to it and keep it the same

Reply:
✅ moved to update_voter_weight_record